open-api.yaml

```diff
@@ -0,0 +1,262 @@
+          example: 49c78036-acfb-11ed-afa1-0242ac120002
+        name:
+          type: string
+          description: Name of ration feedstuff
+          example: Sojabohnen (gesch., Bioprofin)
+        type:
+          type: string
+          description: Feed type of feedstuff in ration, todo check completeness
+          enum: 
+            - dry_forage
+            - corn_silage
+            - alfa_silage
+            - grass_silage
+            - other_wet_forage
+            - by_product
+            - grain
+            - fatty_acid_source
+            - triglycerides
+            - cp_source
+            - rup_source
+            - mineral
+            - miscellaneous
+            - vitamin
+            - premix
+            - forage
+            - concentrate
+            - commercial
+            - water
+          example: dry_forage
+      required:
+        - amount_drymatter
+        - drymatter_percentage
+        - feedstuff_id
+        - name
+        - type
+    RationRequest:
+      title: RationRequest
+      type: object
+      properties:
+        ration_ref:
+          type: string
+          description: client-supplied reference for that ration. If ration_ref is not provided, new ration will be created. If ration_ref is already known, existing ration will be updated.
+          example: 62559890-acfb-11ed-afa1-0242ac120002
+        feedstuffs:
+          type: array
+          description: Order of feedstuff in this array will be used as order of feedstuff in mixer wagon
+          items:
+            $ref: '#/components/schemas/RationFeedstuffRequest'
+        pen_allocations:
+          type: array
+          items:
+            $ref: '#/components/schemas/PenAllocation'
+        name:
+          type: string
+          description: Ration name
+          example: Ration Test 1
+        start_date: 
+          type: string
+          format: date
+          description: Start date of ration, as ISO8601 date
+      required:
+        - feedstuffs
+        - pens
+        - name
+    PenAllocation:
+      title: Pen
+      type: object
+      description: Describes which pen get how much freshmass per animal. Use the number of animals from the animal_data GET /pens interface
+      properties:
+        feed_intake_per_animal:
+          type: number
+          description: Feed intake per animal in freshmass kg per day
+          format: double
+          example: 31.5
+        urn:
+          $ref: "#/components/schemas/URN"
+    RecipeDailyReport:
+      title: RecipeDailyReport
+      type: object
+      properties:
+        loadings:
+          type: array
+          description: List of all loadings per day
+          items:
+            $ref: '#/components/schemas/Loading'
+        ration_ref:
+          type: string
+          description: ration_ref is an optional identifier to connect rations between different systems
+          example: 40ac1b42-acfb-11ed-afa1-0242ac120002
+    Unloading:

```

Comment:
Please indicate which properties are required

Reply:
start_date, pen_urn, unloaded_amount_kg
(end_date and currected_number_of_animals might be implementation specific)